lib/devise-security/validators/password_complexity_validator.rb

```diff
@@ -1,35 +1,47 @@
 # frozen_string_literal: true
 
-# Password complexity validator
-# Options:
-# - digit:  minimum number of digits in the validated string
-# - digits: minimum number of digits in the validated string
-# - lower:  minimum number of lower-case letters in the validated string
-# - symbol: minimum number of punctuation characters or symbols in the validated string
-# - symbols: minimum number of punctuation characters or symbols in the validated string
-# - upper:  minimum number of upper-case letters in the validated string
+# This validator class is used to determine if a password complies with the
+# specified complexity requirements. For example:
+#
+#     validates_with DeviseSecurity::PasswordComplexityValidator,
+#       attributes: :password,
+#       digits: 1, # minimum number of digits in the validated string (also :digit)
+#       lower: 1, # minimum number of lower-case letters in the validated string
+#       symbols: 1, # minimum number of punctuation characters or symbols in the validated string (also :symbol)
+#       upper: 1, # minimum number of upper-case letters in the validated string
```

Comment:
Suggestion to use `@example` tag.

```suggestion
# specified complexity requirements.
#
# @example Validate password has one of each possible type of character
#     attributes: :password,
#     digits: 1, # minimum number of digits in the validated string (also :digit)
#     lower: 1, # minimum number of lower-case letters in the validated string
#     symbols: 1, # minimum number of punctuation characters or symbols in the validated string (also :symbol)
#     upper: 1, # minimum number of upper-case letters in the validated string
#
```

Before:

![Screen Shot 2020-10-04 at 3 11 43 PM](https://user-images.githubusercontent.com/691365/95024945-27d19c00-0654-11eb-8244-c5e18b637636.png)

After:

![Screen Shot 2020-10-04 at 3 11 33 PM](https://user-images.githubusercontent.com/691365/95024951-2c965000-0654-11eb-9c8b-b7e3a31671f8.png)